client/packages/admin/src/index.ts

```diff
@@ -573,11 +573,17 @@ class Storage {
     const headers = {
       ...authorizedHeaders(this.config),
       path,
-      'content-type': metadata.contentType || 'application/octet-stream',
     };
     if (metadata.contentDisposition) {
       headers['content-disposition'] = metadata.contentDisposition;
     }
+    if (!metadata.contentType) {
+      // headers.content-type will become "undefined" (string)
+      // if not removed from the object
+      delete headers['content-type'];
```

Comment:
nit: this is a bit confusing. 

If there _is_ a metadata.contentType
- We overwrite headers['content-type']
But if there _isn't_ a metadata.contentType 
- We _delete_ headers['content-type']

In that case, could we move `delete headers['content-type']` of the if statements?

Reply:
Flipped the logic to 
```ts
    delete headers['content-type'];

    if (metadata.contentType) {
      headers['content-type'] = metadata.contentType;
    }
```